### DISCUSSION TITLE
Why does it ask me for a 'teacher code' to submit answers?

### DISCUSSION THREAD

**Discussion by RonenR** [RonenR/Audactive#2] [Q&A] [answered]
I opened a resource as a student, wanted to submit - but then it required me 'teacher code'. Why? Doesn't it already know who the teacher that made the resource is?

**Comment by RonenR** [comment, marked as answer]
Good question!
Well - the teacher who created the resource - is the Author. 
The author does not wish to get answers from all people in the world.
Also - you as a student don't want to send your answers by mistake to the author, which most chances is NOT YOUR teacher.

So - really? Does it have to be so cumbersome? No. If the teacher shares the lecture - then a unique url that has the teacher's code in it s generated. 
So, when you open a lecture using the teacher's link - you do not need to manually insert the code - it's already there.

**Reply by peterkilcoyne** [reply]
Thanks Ronen
